pkg/kubectl/cmd/taint.go

```diff
@@ -40,6 +40,13 @@ import (
 	utiltaints "k8s.io/kubernetes/pkg/util/taints"
 )
 
+const (
+	MODIFIED    = "modified"
```

Comment:
These should not be here.  
They should either be in a scheduling library or near the api itself.